src/mg_import_csv.cpp

```diff
@@ -717,7 +717,7 @@ int main(int argc, char *argv[]) {
                      .snapshot_wal_mode = memgraph::storage::Config::Durability::SnapshotWalMode::DISABLED,
                      .snapshot_on_exit = true},
       .salient = {.items = {.properties_on_edges = FLAGS_storage_properties_on_edges}}};
-  memgraph::replication::ReplicationState repl_state{memgraph::storage::ReplicationStateRootPath(config)};
+  memgraph::replication::ReplicationState repl_state{memgraph::storage::ReplicationStateRootPath(config), false};
```

Comment:
when we pass here false it is impossible to know what it refers to. If you really need to have bool arg here, at least make it by default false

Reply:
fixed